book/10-git-internals/sections/plumbing-porcelain.asc

```diff
@@ -17,21 +17,22 @@ Here's what it looks like:
 [source,console]
 ----
 $ ls -F1
-HEAD
-config*
+branches/
+config
 description
+HEAD
 hooks/
 info/
 objects/
 refs/
 ----
 
 You may see some other files in there, but this is a fresh `git init` repository – it's what you see by default.
-The `description` file is only used by the GitWeb program, so don't worry about it.
-The `config` file contains your project-specific configuration options, and the `info` directory keeps a global exclude file (((excludes))) for ignored patterns that you don't want to track in a .gitignore file.
+The `description` file is used only by the GitWeb program, so don't worry about it.
+The `config` file contains your project-specific configuration options, and the `info` directory keeps a global exclude file (((excludes))) for ignored patterns that you don't want to track in a `.gitignore` file.
 The `hooks` directory contains your client- or server-side hook scripts, which are discussed in detail in <<_git_hooks>>.
 
-This leaves four important entries: the `HEAD` and (yet to be created) `index` files, and the `objects` and `refs` directories.
+This leaves four important entries: the `HEAD` and (yet to be created) `index` file, and the `objects` and `refs` directories.
```

Comment:
Well, "files" here is referring to "`HEAD` and `index`", so it still makes sense for it to be plural.

Reply:
Ah, quite so, will revert.